spec/Generics.tex

```diff
@@ -277,10 +339,33 @@ \subsection{Formal Arguments of Generic Type}
 \end{chapel}
 specifies that \chpl{tuple.size == 2 && tuple(2).type == real}.
 \end{example}
+
+Homogenous tuple arguments of queried type are also supported:
```

Comment:
To me "queried" (in the above line) means there is a "?". The example below does not have that. Would you reconcile?